docs/operator-guide/deploy-aws-eks.md

```diff
@@ -464,7 +464,11 @@ Please find the detailed description of the variables in the [argo-cd/variables.
 
     :::
 
+## Next Step
+
+  - Install [Argo CD](install-argocd) and integrate it with Edp Cluster Addons
```

Comment:
lets replace "Edp Cluster Addons", for example:
-  Install [Argo CD](install-argocd) and integrate it with [edp-cluster-add-ons](https://github.com/epam/edp-cluster-add-ons/tree/main/argo-cd) components.
or 
- Install [Argo CD](install-argocd) and integrate it with the [edp-cluster-add-ons](https://github.com/epam/edp-cluster-add-ons) repository.